File: First Problem/loopint.cs
using System; 

public class loopint
{
    public static void Main(String[] args)
    {
        int input = 2532;
        Array output = []; 

        string str = input.ToString(); 

        int i = 0; 
        while(i < str.Length)
        {
            int single_output = str[i];
            output.Add(single_output); 
        }

        Console.WriteLine(output); 
    }
}

File: First Problem/loopint.java
import System; 

class loopint
{
    public static void Main(String[] args)
    {
        int num = 2352; 
        int[] arr = new int[4];

        String str = num.toString(); 

        for(int i = 0; str.length(); i++)
        {
            int string = str.charAt(i); 

            arr.push(string); 
        }
        System.out.println(arr);

    }
}
